core/class/jMQTTCmd.class.php

```diff
@@ -246,6 +246,36 @@ public function preSave() {
 
         }
 
+		// Reset auto_publish if info cmd
+		if ($this->getType() == 'info' && $this->getConfiguration('auto_publish', 0))
+			$this->setConfiguration('auto_publish', 0);
+		// Check "request" value if auto_publish enabled
+		if ($this->getType() == 'action' && $this->getConfiguration('auto_publish', 0)) {
+			$req = $this->getConfiguration('request', '');
+			// Get all commands
+			preg_match_all("/#([0-9]*)#/", $req, $matches);
+			$cmds = array_unique($matches[1]);
+			// $value = '';
+			if (count($cmds) > 0) { // There are commands
+				foreach ($cmds as $cmd_id) {
+					$cmd = cmd::byId($cmd_id);
+					if (!is_object($cmd))
+						throw new Exception('Impossible d\'activer la publication automatique sur <b>'.$this->getHumanName().'</b> car la commande <b>'.$cmd_id.'</b> est invalide.');
+					if ($cmd->getType() != 'info')
+						throw new Exception('Impossible d\'activer la publication automatique sur <b>'.$this->getHumanName().'</b> car la commande <b>'.$cmd->getHumanName().'</b> n\'est pas de type info.');
+					if ($cmd->getEqType() =='jMQTT' && $this->getTopic() == $cmd->getTopic())
```

Comment:
should find and remove {toto}{bidule} at the end of the info topic before comparison

Reply:
Indeed, done in next commit